src/components/button-toggle/button-toggle.stories.tsx

```diff
@@ -13,16 +12,14 @@ export const Default: ComponentStory<typeof ButtonToggle> = () => (
 );
 
 export const DefaultWrappedText: ComponentStory<typeof ButtonToggle> = () => (
```

Comment:
comment: this changes the layout of the story

master:
![image](https://github.com/Sage/carbon/assets/44157880/33982194-7df6-46bd-9b9a-f2184b09fc2b)

proposed changes:

![image](https://github.com/Sage/carbon/assets/44157880/514c6a7d-fb9a-42fa-8257-038bb31321d8)